gateway/api/guardrails/guardrails.go

```diff
@@ -40,22 +40,32 @@ func Post(c *gin.Context) {
 		CreatedAt:   time.Now().UTC(),
 		UpdatedAt:   time.Now().UTC(),
 	}
-	err := models.CreateGuardRailRules(rule)
+
+	log.Infof("Creating guardrail %s with connections: %v", rule.Name, req.ConnectionIDs)
```

Comment:
Remove this log statement please, it's not necessary since we're already auditing this endpoint in a middleware